src/common/util/traverse.test.js

```diff
@@ -0,0 +1,57 @@
+import { applyFnToProps } from './traverse'
+
+describe('applyFnToProps', () => {
+  it('should return the object unchanged if it is not an object', () => {
+    const result = applyFnToProps('not an object', jest.fn())
+    expect(result).toEqual('not an object')
+  })
+
+  it('should return the object unchanged if it is null', () => {
+    const result = applyFnToProps(null, jest.fn())
+    expect(result).toBeNull()
+  })
```

Comment:
This could be wrote using `test.each`

```suggestion
  test.each(['not an object', null, undefined])('should return input unchanged when input is not object', (input) => {
    const result = applyFnToProps(input, jest.fn())
    expect(result).toEqual(input)
  })
```